source/frontend/src/features/help/components/HelpModal.tsx

```diff
@@ -57,31 +58,44 @@ const HelpModal: FunctionComponent<React.PropsWithChildren<IModalProps>> = ({
       <ModalStyled dialogAs={DraggableModalDialog} show={show} onHide={handleCancel}>
         <ModalHeader closeButton>
           <DraggableTitle>
-            Help Desk&nbsp;
-            <StyledTooltip toolTipId="help-toolTip" toolTip="Click and drag to move this popup" />
+            <FaQuestionCircle size={24} />
+            <DraggableTitleText>Help Desk</DraggableTitleText>
           </DraggableTitle>
         </ModalHeader>
-
         <Modal.Body>
+          <H3Styled>Get started at PIMS</H3Styled>
+          <p>
+            This overview has useful tools that will support you to start using the application. You
+            can also watch the video demos.
+          </p>
+          <LinkStyled
+            target="_blank"
+            href="https://sp.th.gov.bc.ca/sites/PropertiesServices/Project%20Management/03.%20Execution%20and%20Control/PIMS%20Training"
```

Comment:
I think we discussed this, but I assume this will be replaced with the actual tenant/openshift configmap based url so that we can modify this after deployment without needing a code change?

Reply:
Updated